### ISSUE TITLE
train_loss = inf, valid_loss = inf, time/batch = 0.368

### ISSUE THREAD

**Issue by Instantnoodles-madman** [hardmaru/write-rnn-tensorflow#38]
why loss is inf? how to repair?
